sample-code/javascript-wd/test/helpers/apps.js

```diff
@@ -0,0 +1,8 @@
+if (process.env.DEV) {
```

Comment:
This condition should be switched. I think the idea is that we can run on Sauce? In that case we should be forced to explicitly ask for Sauce stuff (e.g., an environment variable like `SAUCE` set) rather than forcing _all_ local usage to set the `DEV` variable.